src/lib/comms/sol-lwm2m.c

```diff
@@ -1127,6 +1316,124 @@ sol_lwm2m_server_del_registration_monitor(struct sol_lwm2m_server *server,
     return sol_monitors_del(&server->registration, i);
 }
 
+SOL_API struct sol_lwm2m_bootstrap_server *
+sol_lwm2m_bootstrap_server_new(uint16_t port, const char **known_clients)
+{
+    struct sol_lwm2m_bootstrap_server *server;
+    struct sol_network_link_addr servaddr = { .family = SOL_NETWORK_FAMILY_INET6,
+                                              .port = port };
+    int r;
+
+    SOL_LOG_INTERNAL_INIT_ONCE;
+
+    server = calloc(1, sizeof(struct sol_lwm2m_bootstrap_server));
```

Comment:
Ok.

If it's NULL though, I should just terminate the execution, since there's no job to be performed by this Bootstrap Server, right? If this is the case, I think it's better if I check it before allocating memory.